cheroot/connections.py

```diff
@@ -84,9 +100,15 @@ def put(self, conn):
         if conn.rfile.has_data():
             self._readable_conns.append(conn)
         else:
-            self._selector.register(
-                conn.socket.fileno(), selectors.EVENT_READ, data=conn,
-            )
+            self._put_q.append(conn)
+            with self._ctrl_lock:
+                self._ctrl_tx.send(self._CTRL_MSG_PUT)
```

Comment:
Let's dedent this a bit
```suggestion
            return

        # if this conn doesn't have any more data waiting to be read,
        # register it with the selector.
        self._put_q.append(conn)
        with self._ctrl_lock:
            self._ctrl_tx.send(self._CTRL_MSG_PUT)
```